src/ComputationContainer/Share/Compare.cpp

```diff
@@ -184,20 +186,22 @@ bool operator==(const Share<FixedPoint> &left, const FixedPoint &right)
 // Docs/faster-comparison-operators.md
 Share<FixedPoint> LTZ(const Share<FixedPoint> &s)
 {
-    int m = 16;
-    int k = 32;
+    // Experimented and adjusted.
+    int m = 32;
+    int k = 48;
 
-    Share<FixedPoint> x = s * FixedPoint(std::pow(2, m));  // s に 2^m をかけて整数化を試みる
-    Share<FixedPoint> y = FixedPoint(std::pow(2, k)) + x;
+    // s に 2^m をかけて整数化を試みる
+    Share<FixedPoint> x = s * FixedPoint(std::to_string(1LL << m));
+    Share<FixedPoint> y = FixedPoint(std::to_string(1LL << k)) + x;
     Share<FixedPoint> z = getLSBShare(y);
     y = (y - z) * FixedPoint(0.5);
     for (int i = 1; i < k; ++i)
     {
         Share<FixedPoint> b = getLSBShare(y);
-        z += (b * FixedPoint(std::pow(2, i)));
+        z += (b * FixedPoint(std::to_string(1LL << i)));
```

Comment:
When large number is given as the argument of Fixedpoint, the following line caused an overflow.
The string constructor was ok, so I changed to use that one.
https://github.com/acompany-develop/QuickMPC/blob/34ab984a8ec148c0fc9065529e5a3b9bb7e6f3c0/src/ComputationContainer/FixedPoint/FixedPoint.hpp#L71